java/core/src/test/org/apache/orc/impl/TestRecordReaderImpl.java

```diff
@@ -2296,7 +2298,7 @@ public void testPositionalEvolutionAddColumnPPD() throws IOException {
 
     boolean[] rows = applier.pickRowGroups(new ReaderImpl.StripeInformationImpl(stripe,  1, -1, null),
         indexes, null, encodings, null, false);
-    assertEquals(SargApplier.READ_ALL_RGS, rows); //cannot filter for new column, return all rows
+    assertEquals(true, Arrays.equals(SargApplier.READ_ALL_RGS, rows)); //cannot filter for new column, return all rows
```

Comment:
This is irrelevant but satisfies the Travis CI's `EC_BAD_ARRAY_COMPARE` error detection. We can make this as a separate PR.